mtp_cashbook/assets-src/javascripts/app.js

```diff
@@ -17,5 +17,72 @@
   checks.on('change', function() {
     var check = $(this);
     check.closest('tr')[check.is(':checked') ? 'addClass' : 'removeClass']('is-selected');
-  })
+  });
+
+  var tourBase = {
+    i18n: {
+      closeTooltip: 'Dismiss'
+    },
+    onNext: hopscotch.highlight.remove,
+    onEnd: function() {
+      hopscotch.highlight.remove();
+      Cookies.remove('hopscotch.' + tour.id);
+      Cookies.set('hopscotch.state.' + tour.id, 'dismissed');
+    },
+    onClose: function() {
+      var stepNo = hopscotch.getCurrStepNum();
+
+      hopscotch.highlight.remove();
+
+      Cookies.set('hopscotch.state.' + tour.id, 'dismissed');
+
+      if (tour.steps[stepNo].dismissTo) {
+        window.setTimeout(function(){
+          hopscotch.startTour(tour, tour.steps[stepNo].dismissTo);
+        }, 300);
+      }
+    },
+    onShow: function() {
+      var stepNo = hopscotch.getCurrStepNum();
+
+      if (tour.steps[stepNo].highlight) {
+        hopscotch.highlight.show();
+      }
+
+      if (tour.steps[stepNo].fadeout) {
+        window.setTimeout(function(){
+          $('.hopscotch-bubble').fadeOut(300, function(){
+            hopscotch.endTour(false);
+          });
+        }, 5000);
+      }
+
+      if (tour.steps[stepNo].target === 'tour-intro') {
+        $('.hopscotch-bubble-arrow-container').remove();
+        $('.hopscotch-nav-button').text('Yes please');
+        $('.hopscotch-bubble-close').text('Not now');
```

Comment:
That would be good. Although I can't think how to do that right now. Perhaps one for the new dev?